README.md

```diff
@@ -360,6 +361,32 @@ await octokit.rest.issues.create({
 
 Learn more about [how authentication strategies work](https://github.com/octokit/authentication-strategies.js/#how-authentication-strategies-work) or how to [create your own](https://github.com/octokit/authentication-strategies.js/#create-your-own-octokit-authentication-strategy-module).
 
+### Proxy Servers
```

Comment:
I think we should make clear that this is only relevant to Node.js, as Octokit is used in other JS runtime environments such as browsers, Deno, and Cloudflare Workers

```suggestion
### Proxy Servers (Node.js only)
```

Other than that it's a great start, we can iterate as more folks run into problems with proxies.

Thanks!

Reply:
I was thinking about saying something but with a myriad of runtimes I'm not 100% sure what needs it and what doesn't.  Happy to defer that to you though.